DietPi-Software | Node-RED: Switch to a local service user install

MichaIng opened the pull request:
**Status**: Review

**Commit list/description**:
+ DietPi-Software | Node-RED: Switch to a local service user install right into /mnt/dietpi_userdata/node-red/node_modules. This allows Node-RED to control core modules as well. Additionally add a console alias for the admin CLI which invokes the service user instance via sudo. Also do not install Python 2 anymore, everything should now work with Python 3.

MichaIng committed:
v7.0

+ DietPi-Software | Node-RED: Switch to a local service user install right into /mnt/dietpi_userdata/node-red/node_modules. This allows Node-RED to control core modules as well. Additionally add a console alias for the admin CLI which invokes the service user instance via sudo. Also do not install Python 2 anymore, everything should now work with Python 3.

MichaIng commented:
@StephanStS 
Since you actively use Node-RED, could you have a look at this? We previously installed Node-RED as global Node module and command. But that does not allow full control over those initially installed Node-RED modules via its web interface. One can install new modules, update or remove them, but not the ones installed to the global system directories, of course.

It makes much more sense hence to install the modules as "nodered" user right from the start. So they are no globally available Node modules and commands, but everything is in that service users home directory. Also self updates should be possible then, but since there is no update available, I cannot see it currently.

And do you know a reason to keep installing Python 2 together with Node-RED?

And it seems the docs require a fix: https://dietpi.com/docs/software/hardware_projects/#node-red
- It does not run on HTTPS by default
- And `http://<your.IP>:1880/ui` does not exist, `http://<your.IP>:1880` without `/ui` path is right to access the web UI, or do I misunderstand what the dashboard means and is something else required before this is available?

**EDIT: Ah found it: `node-red-dashboard`**
Then we should add an info about how to install that module.
Whoopsie, I merged the docs commit accidentally instead of creating a PR. Please review: https://github.com/MichaIng/DietPi-Docs/commit/5b56e1c39541d8064d68217118ffd5e4b1df00d3

StephanStS commented:
> Whoopsie, I merged the docs commit accidentally instead of creating a PR. Please review: [MichaIng/DietPi-Docs@5b56e1c](https://github.com/MichaIng/DietPi-Docs/commit/5b56e1c39541d8064d68217118ffd5e4b1df00d3)

@MichaIng: I already started to change it also. :-)
My changed were done on https://github.com/StephanStS/DietPi-Docs/tree/dev-StS-NodeRedEditorial. 
The only thing you could do would be to change https -> http at the dashboard ui.

MichaIng committed:
v7.0

+ CHANGELOG | Node-RED: New installs and reinstalls with install Node-RED as local module for the "nodered" service user

MichaIng committed:
v7.0

+ DietPi-Software | Node-RED: npm does not install modules into the users home directory but into the current directory
+ DietPi-Software | Node-RED: Do not remove the local Node-RED install via npm, as the whole directory is removed anyway

MichaIng committed:
v7.0

+ CHANGELOG | Minor wording

MichaIng commented:
I'll merge it now, as I see no issues with it, all web UI elements work perfectly fine.

MichaIng committed:
Merge branch 'dev' into node-red
